.github/workflows/benchmark.yaml

```diff
@@ -1,4 +1,4 @@
-name: Benchmark
+name: Benchmark Jarl
```

Comment:
Will this only run on git push --tags? Or on any push?

Reply:
Tags only. And when manually triggered.